<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>distance_conversion_from_miles_to_km_WIndow</class>
 <widget class="QMainWindow" name="distance_conversion_from_miles_to_km_WIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1 верста = 1066,8 м</string>
    </property>
   </widget>
   <widget class="QLabel" name="distance_conversion_from_miles_to_km_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вычисление пересчета расстояния из верст в километры. </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>351</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="input_frame_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Versta">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Верста  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pounds_input_2">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="distance_conversion_from_miles_to_km_result_button_">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="result_frame_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="result_text_2">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Км  </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
